app/views/candidate_interface/adviser_sign_ups/interruptions/show.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <div class="app-grid-column--grey">
+      <h1 class="govuk-heading-l">You are eligible for a free teacher training adviser</h1>
```

Comment:
I apologise for being a pedant, but is there a reason why this text is here and not in a locale file?